google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -273,9 +273,9 @@ async def list_indexes(
                 maximum=60.0,
                 multiplier=1.3,
                 predicate=retries.if_exception_type(
-                    exceptions.DeadlineExceeded,
```

Comment:
Is the generator going back and forth on the order of these exceptions?  https://github.com/googleapis/gapic-generator-python/blob/b19026d9cca26ebd1cd0c3e73f738c4d1870d987/gapic/templates/%25namespace/%25name_%25version/%25sub/services/%25service/async_client.py.j2#L201-L206

Reply:
I don't understand how that template fragment could be varying the output.